src/WebJobs.Script.WebHost/Resources/Functions/WarmUp/function.json

```diff
@@ -0,0 +1,16 @@
+﻿{
+    "bindings": [
+        {
+            "type": "httpTrigger",
```

Comment:
Making this a full fledged http function exercises the entire http pipeline, which is our goal